site/styles/_shift72.scss

```diff
@@ -2,6 +2,7 @@
 // s72-slider styles in _slider.scss
 // shopping modal styles in _shopping.scss
 // user devices styles in _devices.scss
+@import '_mixins.scss';
```

Comment:
Does this import need to be here?  You didn't put it in the other files?

Reply:
Was getting an error something like mixin doesn't exist in this file and not in the other files. Importing it fixes the issue

Reply:
I think it might be because `_shift72` is imported before `_mixins` in `main.scss`... if you put the mixins import above the shift72 one, you should be able to remove the import from this file.